src/role-access/containers/specific-access/specific-access-information/specific-access-information.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { ErrorMessage } from '../../../../app/models';
+import { ExclusionNavigationEvent, ExclusionState, ExclusionStateData } from '../../../models';
+import { ExclusionNavigation } from '../../../models/exclusion-navigation.interface';
```

Comment:
`ExclusionNavigation` could be moved to the line above?

Reply:
Please re-review. I've changed to specific access navigation now.